src/components/CourseTags/renderSuggestion.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
nit: instead of renderSuggestion, use something along the lines of TagSuggestions, Suggestions, etc.

Reply:
Fixed.